### PR TITLE
CAS-1000 searchinputview customization

### PR DESCRIPTION
**Pull request by samiuelson**
https://stream-io.atlassian.net/browse/CAS-1000

### 🎯 Goal

Allow customizing `SearchInputView` with XML attrs and TransformStyle API

### 🛠 Implementation details

* Added XML attrs 
* Added `SearchInputViewStyle` class
* Added `StreamUi.SearchInputView` theme style allowing global SearchInputView customization using StreamUiTheme

### 🧪 Testing

Override `TransformStyle.searchInputViewStyle` properties or XML attrs of `SearchInputView`.

### ☑️ Checklist

- [x] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required)
- [x] PR targets the `develop` branch
- [x] Changelog is updated with client-facing changes
- [ ] New code is covered by unit tests
- [x] Comparison screenshots added for visual changes
- [x] Affected documentation updated (CMS, cookbooks, tutorial) https://stream-io.atlassian.net/browse/CAS-1107
- [x] Reviewers added

### CODE REVIEW

**Review by adasiewiczr — COMMENTED** [review]
Looks good in overall but we should also allow customizing it via `StreamUiTheme`

**Review by adasiewiczr — APPROVED** [review]

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/TransformStyle.kt [hunk 1]
```diff
@@ -32,6 +33,7 @@ public object TransformStyle {
     public var fileAttachmentStyleTransformer: StyleTransformer<FileAttachmentsViewStyle> = noopTransformer()
     public var suggestionListStyleTransformer: StyleTransformer<SuggestionListViewStyle> = noopTransformer()
     public var messageListHeaderStyleTransformer: StyleTransformer<MessageListHeaderViewStyle> = noopTransformer()
+    public var searchInputViewStyle: StyleTransformer<SearchInputViewStyle> = noopTransformer()
```
Comment (bychkovdmitry) [inline stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/TransformStyle.kt:36]: How about `searchInputViewStyleTransformer` for consistency?

**Review by bychkovdmitry — APPROVED** [review]
Cool, really good

File: stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/TransformStyle.kt (hunk 1 above)
Comment (samiuelson) [reply stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/TransformStyle.kt]: Yup, renamed

**Pull request merged**